encrypted-media-respec.html

```diff
@@ -750,73 +798,63 @@
                 </dl>
               </li>
 
-              <li><p>If <var>implementation</var> in the configuration specified by the combination of the values in <var>accumulated configuration</var> is not supported or not allowed in the <var>origin</var>, return <code>null</code>.</p>
+              <li><p>If <var>implementation</var> in the configuration specified by the combination of the values in <var>accumulated configuration</var> is not supported or not allowed in the <var>origin</var>, return <code>NotSupported</code>.</p>
                 <p class="note">In this step, "supported" includes the implementation being available for use when this algorithm returns, not just user agent support for such an implementation.</p>
               </li>
 
-              <li><p>If <var>accumulated configuration</var>'s <a def-id="option-distinctiveIdentifier"></a> value is <a def-id="requirement-required"></a>:</p>
+              <li>
+                <p>
+                  If <var>accumulated configuration</var>'s <a def-id="option-distinctiveIdentifier"></a> value is <a def-id="requirement-required"></a> and if
+                  the <a def-id="distinctive-identifier"></a> is not <a href="#per-origin-identifiers">unique per-origin</a> and
```

Comment:
This should probably be something like "`...the **implementation's** <a def-id="distinctive-identifier"></a> is not...`" or "`<a def-id="distinctive-identifier">Distinctive Identifier(s)</a> associated with <var>accumulated configuration</var> are not...`"